db-scripts/src/main/resources/migration.sql

```diff
@@ -446,9 +447,37 @@ c1.`CANCER_STUDY_ID` = cancer_study.`CANCER_STUDY_ID` GROUP BY cancer_study.`CAN
 
 UPDATE info SET DB_SCHEMA_VERSION="2.5.0";
 
-##version 2.6.0
+##version: 2.6.0
 -- modify fkc for gistic_to_gene
 ALTER TABLE gistic_to_gene DROP FOREIGN KEY gistic_to_gene_ibfk_2;
 ALTER TABLE gistic_to_gene ADD CONSTRAINT `gistic_to_gene_ibfk_2` FOREIGN KEY (`GISTIC_ROI_ID`) REFERENCES `gistic` (`GISTIC_ROI_ID`) ON DELETE CASCADE;
 
-UPDATE info SET DB_SCHEMA_VERSION="2.6.0";
\ No newline at end of file
+UPDATE info SET DB_SCHEMA_VERSION="2.6.0";
+
+##version: 2.7.0
+
+-- ================== new reference genome foreign key for mutation_event and genetic_profile tables   ==================
+ALTER TABLE `mutation_event` ADD COLUMN `REFERENCE_GENOME_ID` INT(4) NULL AFTER `ENTREZ_GENE_ID`;
+ALTER TABLE `mutation_event`
+    ADD FOREIGN KEY (`REFERENCE_GENOME_ID`) REFERENCES `reference_genome_gene` (`REFERENCE_GENOME_ID`) ON DELETE CASCADE;
+
+UPDATE `mutation_event` set REFERENCE_GENOME_ID = (SELECT `REFERENCE_GENOME_ID` FROM `reference_genome` WHERE BUILD_NAME = 'GRCh37') WHERE NCBI_BUILD in ('hg19', 'GRCh37','37');
+UPDATE `mutation_event` set REFERENCE_GENOME_ID = (SELECT `REFERENCE_GENOME_ID` FROM `reference_genome` WHERE BUILD_NAME = 'GRCh38') WHERE NCBI_BUILD in ('hg38', 'GRCh38');
+UPDATE `mutation_event` set REFERENCE_GENOME_ID = (SELECT `REFERENCE_GENOME_ID` FROM `reference_genome` WHERE BUILD_NAME = 'GRCm38') WHERE NCBI_BUILD in ('mm10', 'GRCm38');
+
+ALTER TABLE `mutation_event` DROP COLUMN `NCBI_BUILD`;
```

Comment:
@khzhu : have you tried importing a new study using metaImport.py tool after dropping this field? So a new study that has not been imported before and that contains new mutations that are not yet in `mutation_event`.  I think this will fail, because the `NCBI_BUILD`  field is dropped here, but I don't see that it is dropped from the dao and model layers used by the importer in this PR.

Reply:
@pieterlukasse , yes, I did import public lgg_unsf_2014 dataset on a migrated database and a fresh created database. The mutation_event table populated correctly.

Reply:
ok, thanks